week-3/Homework/mandatory/3-space-colonies.js

```diff
@@ -8,8 +8,10 @@
   NOTE: don't include any element that is not a "family".
 */
 
-function colonisers() {
-
+function colonisers(arr) {
+  let famArr = arr.filter(x => x.includes("family"));
+  let colonisers = famArr.filter(i => i[0] === "A");
```

Comment:
This works well, may mean that we're going to look through the array twice rather than once; can you think of how to rewrite this code to only be one filter rather than two?

Reply:
I found this article and decided to try to experiment and compare "as is" with % and " and received positive results. later on the call teachers found some exceptions and why this may not work in certain cases, but i was very busy with exercise 7 and did not have time to come back and do a better version...